src/pages/index.js

```diff
@@ -60,28 +107,122 @@ document.addEventListener("DOMContentLoaded", function () {
     addFormValidator.resetValidation();
   });
 
-  newCardList.renderItems();
+  constants.avatar.addEventListener("click", () => {
+    avatarPopup.open();
+    addFormValidator.resetValidation();
```

Comment:
`    addFormValidator.resetValidation();
`

`addFormValidator` is for the card form but you open here the avatar form

Reply:
corrected. Thank you.